tests/calc/test_thermo.py

```diff
@@ -22,6 +22,7 @@
                         relative_humidity_from_mixing_ratio,
                         relative_humidity_from_specific_humidity,
                         relative_humidity_wet_psychrometric,
+                        gradient_richardson_number,
```

Comment:
Can you put this up between `exner_function` and `isentropic_interpolation` so that the list stays alphabetized?

Reply:
Done!